src/main/java/com/sonyericsson/rebuild/RebuildAction.java

```diff
@@ -296,20 +296,13 @@ public void doConfigSubmit(StaplerRequest req, StaplerResponse rsp) throws Servl
                 }
             }
             for (ParameterValue source : paramAction.getParameters()) {
-                boolean alreadyAdded = false;
-                for (ParameterValue dest : values) {
-                    if (source.getName().equals(dest.getName())) {
-                        alreadyAdded = true;
-                        break;
-                    }
-                }
-                if (!alreadyAdded) {
+                if (!values.contains(source)) {
```

Comment:
Not sure it will work properly for all `ParameterValue ` implementations.

Reply:
I was unsure why the previous code was so roundabout but that makes sense why it is. Reverted this bit.